en/blazor-support.md

```diff
@@ -52,12 +52,38 @@ Generating Blazor application has the following limitations:
 - Bootstrap is turned off in the generated application for now.
 
 Generated components have these limitations:
-- Navigation drawer works only in the overlay state. The mini version does not work too.
-- Date picker works only in dropdown mode.
-- The input group does not support date type.
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
+- Navigation Drawer - does not support pin state and pin threshold.
+- Avatar:
+    - does not use IgbIcon in generated code.

```

Comment:
How about we put a single global note about all icons using simple Material Icons font content instead of a full `IgbIcon` instead of putting this note on every component that uses icons?

Also is this the reason why the Avatar is marked as partial in the supported components table? I'm inclined to mark it as fully functional actually, there's nothing wrong with the Avatar itself and it works just fine for the features we expose, right? It's the icon inside, which is global and separate, sort of.

PS: Also do mark code references such as component classes with `` ` ``s if you're going to use the names like that

Reply:
> Also is this the reason why the Avatar is marked as partial in the supported components table? I'm inclined to mark it as fully functional actually, there's nothing wrong with the Avatar itself and it works just fine for the features we expose, right? 

When set a string with length bigger than two letters in Blazor Avatar it is trying to show entire string instead of just first two letters. In AppBuilde and in Angular generated application only first two letters are shown.